cdap-client-tests/src/test/java/co/cask/cdap/client/MetadataHttpHandlerTestRun.java

```diff
@@ -158,7 +158,7 @@ public void testProperties() throws Exception {
 
     // test search for application
     Set<MetadataSearchResultRecord> expected = ImmutableSet.of(
-      new MetadataSearchResultRecord(application)
+      new MetadataSearchResultRecord(application.toEntityId())
```

Comment:
check if ``application``, ``mystream``, ``myview``, etc in this class can be the respective newer, ``EntityId`` class